.github/workflows/ci.yml

```diff
@@ -16,8 +16,8 @@ jobs:
       matrix:
         include:
           - pair:
-              elixir: 1.12.3
-              otp: 24.3.4
+              elixir: 1.13.4
+              otp: 24.3.4.17
```

Comment:
@vanjacosic Can we support Elixir 1.17.x and Erlang OTP 27.x in the CI pipeline?